modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/organization_controller.rb

```diff
@@ -15,14 +15,16 @@ def submit
           shared_form_validation(FORM_NUMBER)
           poa_code = get_poa_code(FORM_NUMBER)
           validate_org_poa_code!(poa_code)
+          rep = validate_registration_number!
 
-          submit_power_of_attorney(poa_code, FORM_NUMBER)
+          submit_power_of_attorney(poa_code, FORM_NUMBER, rep)
         end
 
         def validate
           shared_form_validation(FORM_NUMBER)
           poa_code = get_poa_code(FORM_NUMBER)
           validate_org_poa_code!(poa_code)
+          validate_registration_number!
```

Comment:
can we not put this in the shared validation and return the rep at :15, for use in :20?